src/index.js

```diff
@@ -184,3 +191,11 @@ export default class StickyBox extends React.Component {
     );
   }
 }
+
+StickyBox.defaultProps = {
+  onChangeMode: () => {},
```

Comment:
there already is a `defaultProps` function in line 31,
it probably makes sense to move this, as well as the propTypes definition up there. (Feel free to add PropType definitions for `bottom: boolean, and offset: number` there as well :)